### ISSUE TITLE
It doesn't work if there are no tags at all yet

### ISSUE THREAD

**Issue by fregante** [fregante/daily-version-action#18]
```
Error: Error Command failed: git fetch --depth=1 origin refs/tags/*:refs/tags/*
```

**Comment by i-am-the-slime** [comment]
Can you say what the workaround for this is? Does the tag need a specific name?
EDIT:  Looks like creating any tag and pushing it is enough

**Comment by NachoJusticia** [comment]
@i-am-the-slime you are right, creating any tag makes it work fine

**Comment by i-am-the-slime** [comment]
@NachoJusticia Thanks! I know I'm right 😆

**Comment by tiqwab** [comment]
Does `git fetch --depth=1 --tags origin` work in this case? It fetches all tags as well as the latest HEAD commit, so should work almost same as the original command. 

(but I realized that you changed to specify refspec instead of `--tags` in https://github.com/fregante/daily-version-action/pull/4, so something wrong with `--tags`?)

I changed the command in https://github.com/tiqwab/daily-version-action/tree/fix-fetch-tag-command and it looks like working well.

<details>
<summary>Test summary</summary>

### Workflow for test

```yml
name: Test daily-version

on:
  workflow_dispatch:

jobs:
  Tag:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2

      # Use current one or edited one
      - uses: tiqwab/daily-version-action@v1
      # - uses: tiqwab/daily-version-action@234e981a452101e7ded94b3dd66513fb69d7d237
        id: daily-version
    outputs:
      created: ${{ steps.daily-version.outputs.created }}
      version: ${{ steps.daily-version.outputs.version }}

  Next:
    needs: Tag
    if: needs.Tag.outputs.created
    runs-on: ubuntu-latest
    steps:
      - run: echo It looks like ${{ needs.Tag.outputs.version }} was created!

```

### Result

- when use `git fetch --depth=1 origin refs/tags/*:refs/tags/*`
  - there is no tag
    - it fails: `Error Command failed: git fetch --depth=1 origin refs/tags/*:refs/tags/*`
  - there is a tag and it is on HEAD
    - it works: no tag is created
  - there is a tag but it is not on HEAD
    - it works: new tag is created
- when use `git fetch --depth=1 --tags origin`
  - there is no tag
    - it works: new tag is created
  - there is a tag and it is on HEAD
    - it works: no tag is created
  - there is a tag but it is not on HEAD
    - it works: new tag is created
</details>

**Comment by fregante** [comment]
@tiqwab I don't have time to research why I made that change in https://github.com/fregante/daily-version-action/commit/f2d9b7491afcdd29613e03de6f36b0c7b3de434e, issue https://github.com/fregante/daily-version-action/issues/3 might have the explanation.

I sent a PR to just ignore the fetch error. 👍

**Comment by fregante** [comment]
Sorry for the many people bitten by this bug, the solution was in front of us this whole time. Fixed and released in the action’s v2.0.0

**Comment by tiqwab** [comment]
I agree that it is easy and reasonable way to handle it, thanks!
